lldb/source/Commands/CommandObjectDWIMPrint.h

```diff
@@ -43,6 +43,10 @@ class CommandObjectDWIMPrint : public CommandObjectRaw {
   HandleArgumentCompletion(CompletionRequest &request,
                            OptionElementVector &opt_element_vector) override;
 
+protected:
+  /// Stores the expression after applying any Fix-Its.
+  std::string m_fixed_expression;
```

Comment:
It appears this doesn't need to be a member variable, that it can be a local variable in `CommandObjectDWIMPrint::DoExecute`.

Reply:
Sure thing.

I was wondering what that would be better between using a local versus being consistent with `CommandObjectExpression` class.

Reply:
Yes I noticed that expression needs it to be a member variable. Personally I'd prefer to see this class be slightly different (it is already, anyway), than copy what expression does. It's good to keep them consistent, but it's better, imo, to avoid introducing bits of unnecessary complexity.

Reply:
Now fixed as a local variable.